crates/burn-jit/src/kernel/conv/conv2d/direct.rs

```diff
@@ -50,7 +49,7 @@ fn conv2d_kernel<F: Float>(
     let oh = ABSOLUTE_POS / output.stride(2) % output.shape(2);
     let ow = ABSOLUTE_POS / output.stride(3) % output.shape(3);
 
-    let g = (weight.shape(0) + oc) % args.groups;
+    let g = oc / args.channels_per_group;
```

Comment:
Is this the fix for the bug you mentioned about not having the exact same result as torch?

Reply:
Exactly.